tools/dgeni/src/processors/addInheritedDocsContent.ts

```diff
@@ -0,0 +1,26 @@
+import { Processor, Document } from 'dgeni';
+
+/**
+ * Inherit docs content from parent interfaces.
+ */
+export class AddInheritedDocsContentProcessor implements Processor {
+	name = 'filterOutPrivateProperties';
+	$runAfter = ['docs-processed'];
+	$runBefore = ['rendering-docs'];
+
+	$process(docs: Document[]): Document[] {
+		return docs.map(doc => {
+			if(!doc.members || !doc.tags.tags.filter(tag => tag.tagName === 'inheritdoc').length) return doc;
+
+			//get all interfaces that the doc implements.
+			doc.moduleDoc.exports.filter(e => e.symbol.escapedName.includes('Interface'))
```

Comment:
Why `.includes` instead of a strict equality check?

Reply:
That field would be something like "DaffCartFacadeInterface"

Reply:
Is there a prop on a dgeni doc that gives the symbols type which could be compared against? 

While today we use `*Interface` as the convention, I can imagine various scenarios where a different convention gets used. 

It would be nice to allow for such an edge-case.

Reply:
I'll take a look

Reply:
changed